HHS/Head-Start-TTADP
# [TTAHUB-2683] Use status change service fn in place of `Goal.update({ status })`

## Description of change

In this PR we begin using the `changeGoalStatus` helper in place of e.g. `Goal.update({ status })`. It's only a few places, it seems.


## How to test

- Make sure the tests pass and the code makes sense.
- Ensure there are no other places I missed where we change goal status.

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2683


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
